tests/queries/0_stateless/03312_issue_74299.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CUR_DIR"/../shell_config.sh
+
+$CLICKHOUSE_CLIENT --query="CREATE TABLE t0 (c0 Int) engine=Memory"
+$CLICKHOUSE_CLIENT --query="INSERT INTO t0 values (1)"
+($CLICKHOUSE_CLIENT --query="INSERT INTO TABLE t0 (c0) VALUES (INTERVAL currentProfiles() MINUTE)" 2>/dev/null && echo "Expected failure") || true
+$CLICKHOUSE_CLIENT --query="SELECT * FROM t0 where 1"
```

Comment:
Cannot this be written in a single sql script?

```
CREATE TABLE t0 (c0 Int) engine=Memory;

INSERT INTO t0 values (1);
INSERT INTO TABLE t0 (c0) VALUES (INTERVAL currentProfiles() MINUTE);
SELECT * FROM t0 where 1;
```

SQL scripts are more favorable then SH scripts, because they are simpler, faster and our fuzzer can use them as seed queries.

Reply:
Insert test should fail during type checking and return an error code. This could cause a fast test failure, so I think we should fall back to using a shell script.

Reply:
FWIW I think that you should still be able to do the same using a sql script and be able assert for the error code using error tags like [this](https://github.com/ClickHouse/ClickHouse/blob/master/tests/queries/0_stateless/02981_variant_type_function.sql#L9)  (for an example).

Reply:
Fall back to sql file. I didn't know the appearance of error tags until seeing TestHint.
Thanks for your help.